projects/ngx-oxford/src/lib/oxford.pipe.ts

```diff
@@ -31,9 +45,16 @@ export class OxfordPipe implements PipeTransform {
     for (let i = 0; i < trimmedValues.length - 1; i++) {
       result += `${OxfordPipe.trim(trimmedValues[i])}, `;
     }
-    result += `${OxfordPipe.trim(conjunction)} ${OxfordPipe.trim(
-      trimmedValues[trimmedValues.length - 1],
-    )}`;
+    if (!truncate) {
+      result += `${OxfordPipe.trim(conjunction)} ${OxfordPipe.trim(
+        trimmedValues[trimmedValues.length - 1],
+      )}`;
+    } else if (truncate.maxWords > values.length) {
```

Comment:
One last thing and then I can merge this in! This conditional block should still call `OxfordPipe.trim` on `trimmedValues[trimmedValues.length - 1]` to prevent extra space. It's really the same as the first condition of `!truncate`.

Probably best to just combine them into a single block like `if (!truncate || truncate.maxWords > values.length) {...} else {...}`